<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotepadWindow</class>
 <widget class="QMainWindow" name="NotepadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Incybro Writer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>581</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalics"/>
   <addaction name="actionUnderline"/>
   <addaction name="actionLeft"/>
   <addaction name="actionCenter"/>
   <addaction name="actionRight"/>
   <addaction name="actionJustify"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>609</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/undo.png</normaloff>:/Images/img/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/redo.png</normaloff>:/Images/img/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/cut.png</normaloff>:/Images/img/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/copy.png</normaloff>:/Images/img/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/paste.png</normaloff>:/Images/img/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/new.png</normaloff>:/Images/img/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/open.png</normaloff>:/Images/img/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/save.png</normaloff>:/Images/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/saveas.png</normaloff>:/Images/img/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/Bold.png</normaloff>:/Images/img/Bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/left.png</normaloff>:/Images/img/left.png</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionCenter">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/center.png</normaloff>:/Images/img/center.png</iconset>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/right.png</normaloff>:/Images/img/right.png</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionJustify">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/justify.png</normaloff>:/Images/img/justify.png</iconset>
   </property>
   <property name="text">
    <string>Justify</string>
   </property>
   <property name="toolTip">
    <string>Justify</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/underline.png</normaloff>:/Images/img/underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="toolTip">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionItalics">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/italics.png</normaloff>:/Images/img/italics.png</iconset>
   </property>
   <property name="text">
    <string>Italics</string>
   </property>
   <property name="toolTip">
    <string>Italics</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/Images/img/print.png</normaloff>:/Images/img/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
